mu-plugins/central-hub/src/config-store/internals.php

```diff
@@ -69,6 +69,11 @@ function _the_store( $store_key = '', $config_to_store = array(), $remove = null
  * @return array
  */
 function _load_config_from_filesystem( $path_to_file ) {
+
+	if ( ! $path_to_file ) {
```

Comment:
@rgadon107 This conditional isn't necessary.  Why?  The `$path_to_file` is a required parameter.  When you invoke the function, you have to pass an argument to it.  If the value received is a falsey, such as `false`, `null`, etc., an error will be thrown when the `require` runs.

Reply:
@hellofromtonya This conditional was removed on 05-10-19 following the discussion in our code review.